net-mail/b4/b4-0.10.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{8,9,10} )
```

Comment:
I think the issue is that we've kept Python 3.8 here but no versions of git-filter-repo (a dependency of b4) have Python 3.8 support.

Reply:
WFM, I've squashed in a fix to drop 3.8.